auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneNumberUtils.java

```diff
@@ -93,6 +88,14 @@ static String formatPhoneNumber(@NonNull String phoneNumber, @NonNull CountryInf
     static String formatPhoneNumberUsingCurrentCountry(
             @NonNull String phoneNumber, Context context) {
         final CountryInfo currentCountry = PhoneNumberUtils.getCurrentCountryInfo(context);
+
+        // TODO(samstern): Remove this call once the next version of Play servics is released
+        //                 estimated timelime August 10th
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Might be worth adding a (known bug) to the commend that hint selector is still susceptible to the false negatives of this method.